orion/sites/admin/src/announce/announce.js

```diff
@@ -3,11 +3,66 @@ require('./announce.styl');
 import React from 'react';
 import ReactDOM from 'react-dom';
 
+
 export class AnnouncePage extends React.Component {
 	render() {
+		var numAnnouncements = 5; 	{/* this needs to be a variable later */}
+
+		{/* these fake announcements will eventually be replaced with real announcements */}
+		var fakeAnnounceA = {
```

Comment:
the new javascript way (esp in react) is to use `let fakeAnnounceA = ...` instead of var.